### DISCUSSION TITLE
Difference between the preview.organized-app.com site and the organized-app.com site?

### DISCUSSION THREAD

**Discussion by luisenriquereyes** [sws2apps/organized-app#2937] [Q&A] [answered]
What is the difference between creating a login and congregation on the preview.organized-app.com site versus the organized-app.com site?

**Comment by fschein** [comment]
Great question. I would like to know, too.

**Comment by ux-git** [comment, marked as answer]
Thank you for such a great question!

The difference between these links is mainly in their purpose and data handling:

• preview.organized-app.com is used for previewing and developing the upcoming release. It’s a development environment with a temporary database that is auto-cleared periodically. This allows developers to test and refine new features before the public release. After the app release its use might be reconsidered, let's see if we'll need it at all.

• organized-app.com is the main, live app site currently hosting the old version. When the new version is ready, it will be moved here, becoming the primary app for all users.

Additionally, we have:

• test.organized-app.com, a testing environment where users can explore how the Organized app works without needing to register. It uses pre-filled dummy data for congregations, and any changes made will reset after refreshing the page.

So as a conclusion, use the test environment link to check out how Organized works and whether it's a good fit for you, and if so – use the main app link to actually start using the real Organized app (after the release, soon).

Hope this clears up the differences!
